src/oc/code.cpp

```diff
@@ -437,32 +437,6 @@ void hoc_init_space() {
 static int maxinitfcns;
 static Pfrv initfcns[MAXINITFCNS];
 
-/** @brief Print up to the 10 most-recently-pushed elements on the stack.
- */
-void hoc_prstack() {
```

Comment:
Can we leave this one please? It is useful for debugging...

Reply:
I put it back.